mmv1/api/resource.go

```diff
@@ -1217,20 +1217,30 @@ func (r Resource) GetIdFormat() string {
 }
 
 // Returns true if the Type is in the ID format and false otherwise.
-func (r Resource) InIdFormat(prop Type) bool {
-	fields := r.ExtractIdentifiers(r.GetIdFormat())
-	return slices.Contains(fields, google.Underscore(prop.Name))
+func (r Resource) InPostCreateComputed(prop Type) bool {
```

Comment:
I am not sure I understand the name of this new function. InIdFormat made sense because it was checking presence of field in ids. This also seems to do the same thing?